cf-editor-form-field-customization-sample/src/aem-cf-editor-1/web-src/src/components/CustomizedField.js

```diff
@@ -92,16 +79,17 @@ const CustomizedField = () => {
             name={fieldData.name}
             maxLength={fieldData.maxLength}
             onChange={onChangeHandler}
-            validationState={(validationState === "valid" && isValid) ? "valid" : "invalid"}
```

Comment:
We need to check the parent `validationState` status to provide a visual feedback to customers.